apps/server/lib/utils/accounts.ts

```
@@ -0,0 +1,47 @@
+import { auth } from "@nimbus/auth/auth";
+import { db } from "@nimbus/db";
+
+export class AccountError extends Error {
+	constructor(
+		message: string,
+		public code: string = "ACCOUNT_ERROR"
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)